tests/goth_tests/test_run_ssh.py

```diff
@@ -0,0 +1,82 @@
+"""An integration test scenario that runs the Simple Service example requestor app."""
+import asyncio
+import logging
+import os
+from pathlib import Path
+import signal
+import time
+from typing import List
+
+import pytest
+
+from goth.configuration import load_yaml, Override
+from goth.runner.log import configure_logging
+from goth.runner import Runner
+from goth.runner.probe import RequestorProbe
+
+from .assertions import assert_no_errors, assert_all_invoices_accepted
+
+
+logger = logging.getLogger("goth.test.run_ssh")
+
+SUBNET_TAG = "goth"
+
+
+@pytest.mark.asyncio
+async def test_run_ssh(
+    log_dir: Path,
+    project_dir: Path,
+    goth_config_path: Path,
+    config_overrides: List[Override],
+) -> None:
+
+    configure_logging(log_dir)
+
+    # This is the default configuration with 2 wasm/VM providers
+    goth_config = load_yaml(goth_config_path, config_overrides)
+
+    requestor_path = project_dir / "examples" / "ssh" / "ssh.py"
+
+    runner = Runner(
+        base_log_dir=log_dir,
+        compose_config=goth_config.compose_config,
+    )
+
+    async with runner(goth_config.containers):
+
+        requestor = runner.get_probes(probe_type=RequestorProbe)[0]
+
+        async with requestor.run_command_on_host(
+            f"{requestor_path} --subnet-tag {SUBNET_TAG}",
+            env=os.environ,
+            get_process_monitor=True,
+        ) as (_cmd_task, cmd_monitor, process_container):
+            start_time = time.time()
+
+            def elapsed_time():
+                return f"time: {(time.time() - start_time):.1f}"
+
+            cmd_monitor.add_assertion(assert_no_errors)
+            cmd_monitor.add_assertion(assert_all_invoices_accepted)
+
+            # # A longer timeout to account for downloading a VM image
```

Comment:
I don't understand this comment.
Is there any reason why we should have a long timeout in this particular test (and not e.g. in `test_run_blender.py`)?

Reply:
I believe there is such a timeout in the blender test, just no such comment there

Reply:
I copied the base script (and the comment) from the simple service test, that's why it made its way here